app/scripts/session/sessionServices.js

```diff
@@ -140,22 +107,22 @@ window.vcSession = window.vcSession ? window.vcSession : {};
       },
 
       getVirtualCohortDetails: function(virtualCohortID) {
-       /* $.getJSON( vcSession.URL + virtualCohortID, function(response) {
-          iViz.applyVC(_.omit(response.data.virtualCohort,
-            ['created', 'userID', 'virtualCohortID']));
-          jQuery.notify('Imported Virtual Cohort', 'success');
-        }).fail(function() {
-          var virtualCohort_ = _.findWhere(vcSession.utils.getVirtualCohorts(), {
-            virtualCohortID: virtualCohortID
-          });
-          if (virtualCohort_ !== undefined) {
-            iViz.applyVC(_.omit(virtualCohort_,
-              ['created', 'userID', 'virtualCohortID']));
-            jQuery.notify('Imported Virtual Cohort', 'success');
-          } else {
-            jQuery.notify('Error While importing Virtual Cohort', 'error');
-          }
-        });*/
+        /* $.getJSON( vcSession.URL + virtualCohortID, function(response) {
```

Comment:
Sure. Kept it because in future if we want to test visualizing a virtual cohort in iViz.